<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>local_master</class>
    <widget class="QMainWindow" name="local_master">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>480</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>主机</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>130</x>
                        <y>170</y>
                        <width>171</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">font: 25pt &quot;8514oem&quot;;</string>
                </property>
                <property name="text">
                    <string>等待对方玩家进入</string>
                </property>
            </widget>
            <widget class="QPushButton" name="chess1">
                <property name="geometry">
                    <rect>
                        <x>60</x>
                        <y>460</y>
                        <width>40</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="text">
                    <string>石头</string>
                </property>
            </widget>
            <widget class="QPushButton" name="chess2">
                <property name="geometry">
                    <rect>
                        <x>190</x>
                        <y>460</y>
                        <width>40</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="text">
                    <string>剪刀</string>
                </property>
            </widget>
            <widget class="QPushButton" name="chess3">
                <property name="geometry">
                    <rect>
                        <x>320</x>
                        <y>460</y>
                        <width>40</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="text">
                    <string>布</string>
                </property>
            </widget>
            <widget class="QTextEdit" name="textEdit">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>60</y>
                        <width>104</width>
                        <height>71</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">font: 25pt &quot;8514oem&quot;;</string>
                </property>
            </widget>
            <widget class="QLabel" name="num_chess1">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>460</y>
                        <width>40</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">border:1px solid black;
                        background-color: rgb(255, 255, 255);
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="num_chess2">
                <property name="geometry">
                    <rect>
                        <x>230</x>
                        <y>460</y>
                        <width>40</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">border:1px solid black;
                        background-color: rgb(255, 255, 255);
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="num_chess3">
                <property name="geometry">
                    <rect>
                        <x>360</x>
                        <y>460</y>
                        <width>40</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">border:1px solid black;
                        background-color: rgb(255, 255, 255);
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>480</width>
                    <height>23</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
